robustcontrol/utils.py

```diff
@@ -1721,7 +1749,36 @@ def feedback_mimo(G, K=None, positive=False):
 #                                Chapter 4                                    #
 ###############################################################################
 
+def min_max_sigma(G, w, minimum=True, maximum=False):
```

Comment:
I feel like someone could be tempted to set minum=True and maximum=True in this way of calling. Can't we just have a `sense` parameter is default `"min"` but can also be set to `"max"` that way the calling convention is less ambiguous.

Reply:
Changed.